src/main.ts

```diff
@@ -2,25 +2,25 @@ import * as core from '@actions/core';
 import * as tc from '@actions/tool-cache';
 import * as path from 'path';
 import * as fs from 'fs';
+import * as exec from '@actions/exec';
 
 async function run() {
   try {
 
-    // Tripple check it's Windows process
-    // Can't install nuget.exe for Ubuntu image etc..
-    const IS_WINDOWS = process.platform === 'win32';
-    if(IS_WINDOWS === false){
-      core.setFailed("Nuget.exe only works for Windows.");
-      return;
+    // Download mono when linux (macos already supports mono)
+    if (process.platform == 'linux') {
+      await exec.exec('sudo apt-get install mono-complete');
```

Comment:
Do we need to do this if the Ubuntu hosted images have MONO on them?
https://github.com/actions/virtual-environments/blob/master/images/linux/Ubuntu2004-README.md